docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,43 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-iso-image]]
+== Choosing the ISO image
+
+When you start Minishift, it downloads a live ISO image that the hypervisor uses to
+provision the host VM.
+
+The following ISO images are available:
+
+- link:https://github.com/minishift/minishift-b2d-iso/releases[Boot2Docker] (Default). This ISO is
+based on link:https://github.com/boot2docker/boot2docker[Boot2Docker], which is a
+lightweight Linux distribution optimized for containerized application development. This ISO image
+size is small and optimized for development but not for production.
```

Comment:
The word `optimized` and `development` is repeated and it is not looking good at the end. Also `optimized for containerized application development` is technically not correct as Boot2Docker was just created to run Docker containers. Refer [1]. That's the reason I have avoided these strings when I suggested it last time. 
[1] http://boot2docker.io/